src/components/entityManager/EntityManager.tsx

```diff
@@ -290,7 +272,7 @@ export default function EntityManager<E extends Record<string, EntityValueType>>
           <TableCell colSpan={2}>
             <Input
```

Comment:
Can we have the class style for the full name <Input> to be as no border  (or a very subtle one) as in normal condition and when we start editing (that means when it gets focus), the border can appear? This way the field will look  more similar to other, I think